ngraph/frontend/onnx_import/src/op/mean_variance_normalization.cpp

```diff
@@ -49,11 +49,14 @@ namespace ngraph
                 OutputVector mean_variance_normalization(const Node& node)
                 {
                     auto data = node.get_ng_inputs().at(0);
-                    auto axes = node.get_attribute_value<std::vector<int64_t>>("axes", {0, 2, 3});
+                    auto axes =
+                        node.get_attribute_value<std::vector<std::int64_t>>("axes", {0, 2, 3});
                     const std::vector<std::size_t> normalized_axes = ngraph::normalize_axes(
                         node.get_description(), axes, data.get_partial_shape().rank());
-
-                    return {std::make_shared<default_opset::MVN>(data, AxisSet(normalized_axes))};
+                    auto const_axes = default_opset::Constant::create(
+                        element::i64, Shape{normalized_axes.size()}, normalized_axes);
+                    return {std::make_shared<ngraph::op::v6::MVN>(
```

Comment:
Why don't we use `default_opset`? I think that ONNX importer should be switched to opset6.

Reply:
We are working on switching default opset  to version 6 in this sprint. It will be done in a separate PR.

Reply:
task 47584 is currently in progress.